packages/airnode-node/src/constants.test.ts

```diff
@@ -23,5 +23,6 @@ describe('constants', () => {
     expect(constants.PRIORITY_FEE).toEqual(3120000000);
     expect(constants.BASE_FEE_MULTIPLIER).toEqual(2);
     expect(constants.MAXIMUM_ONCHAIN_ERROR_LENGTH).toEqual(100);
+    expect(constants.PROCESSING_TIMEOUT).toEqual(10_000);
```

Comment:
Processing timeout should really be configurable from `config.json` as it can be much longer than 500ms when relying on remote I/O.